examples/protein-neq-switching/README.md

```diff
@@ -0,0 +1,36 @@
+# Run neq switching for a protein mutation (ALA->ASP dipeptide)
+1. First generate a pickled htf. 
+An example htf is present in `2/`. The code to generate one:
+```python
+import pickle
+import os
+from perses.app.relative_point_mutation_setup import PointMutationExecutor
+from simtk import unit
+
+solvent_delivery = PointMutationExecutor("ala_vacuum.pdb",
+                        '1',
+                        '2',
+                        'ASP',
+                        ionic_strength=0.15*unit.molar,
+                        flatten_torsions=True,
+                        flatten_exceptions=True, 
+                        conduct_endstate_validation=False
+                       )
+apo_htf = solvent_delivery.get_apo_htf()
+with open(os.path.join(outdir, f"2_solvent.pickle"), "wb") as f:
+    pickle.dump(apo_htf, f)
+
+```
+2. Run neq switching: (copied from a bash script)
+```bash
+old_aa="ala"
+new_aa="asp"
+resid="2"
+outdir="2/"
+phase='solvent'
+length=1 # in ns to run eq and neq
+
+python run_neq_distrib_flattened.py $outdir $phase "$((${LSB_JOBINDEX}-1))" $old_aa $new_aa $length
```

Comment:
What should `"$((${LSB_JOBINDEX}-1))"` be if someone is running this example locally?

Reply:
It looks like that script is looking for different args 
```
usage: run_neq_distrib_flattened.py [-h]
                                    dir phase sim_number eq_length neq_length
run_neq_distrib_flattened.py: error: the following arguments are required: dir, phase, sim_number, eq_length, neq_length
```
since it doesn't seem to be asking for old_aa or new_aa

Reply:
its the simulation number

Reply:
normally we run 100 jobs in parallel so, `sim_number` corresponds to the job number if using a job array

Reply:
so if you want to just run one job locally, you can just feed any number (e.g. 1)

Reply:
```
$ python run_neq_distrib_flattened.py 2/ solvent 1 ala asp 1
usage: run_neq_distrib_flattened.py [-h]
                                    dir phase sim_number eq_length neq_length
run_neq_distrib_flattened.py: error: argument eq_length: invalid float value: 'ala'
```

Reply:
based on that error, all you need to do is change 
```
python run_neq_distrib_flattened.py 2/ solvent 1 ala asp 1
``` 
to 
```
python run_neq_distrib_flattened.py 2/ solvent 1 1
```

Reply:
Okay so is the bit of example code in the README wrong then?

Reply:
Fixed!